string config_name
---
bool result
